src/Avalonia.Controls/SystemDialog.cs

```diff
@@ -52,6 +52,11 @@ public class SaveFileDialog : FileDialog
         /// </summary>
         public string? DefaultExtension { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to display a warning if the user specifies the name of a file that already exists.
+        /// </summary>
+        public bool? OverwritePrompt { get; set; }
```

Comment:
I think the name needs to change:

`IsOverwritePromptShown` or
`ShowOverwritePrompt`

The name `OverwritePrompt` doesn't work for a bool value and implies that you are setting the prompt object itself.

Reply:
Addressed.

Reply:
Note that this deviates from WPF but I think its justified. 

https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.savefiledialog.overwriteprompt?view=windowsdesktop-6.0